pkg/policy/repository_test.go

```diff
@@ -1902,8 +1923,7 @@ Resolving ingress policy for [any:bar]
     Allows from labels {"matchLabels":{"any:baz":""},"matchExpressions":[{"key":"any:baz","operator":"In","values":[""]}]}
       No label match for [any:foo]
 * Rule {"matchLabels":{"any:bar":""}}: selected
-    Enforcing requirements [{Key:any.baz Operator:In Values:[]}]
-      No label match for [any:foo]
+      Found all required labels
```

Comment:
This was the part I was trying to avoid ;)

What does this mean?

Reply:
The problem here is that a rule with just `FromRequires` will never allow any traffic by itself. If we add the requirements (collected from this same rule) to this rule as MatchExpressions, then this rule will start allowing traffic from endpoints matching the requirement. Fix might be as simple as logging "Rule does not allow any traffic" when `fromEndpoints` is empty, instead of the "Found all required labels"?

Reply:
Or just print out the "Found all required labels" when there are fromEndpoints to match?

Reply:
I separated this fix to a new (first) commit, please take a look. Now a rule that does not allow anything will not print any trace output. Maybe we should print out a note that the "Rule does not allow any traffic" etc.?